src/python/build.sh

```diff
@@ -0,0 +1,30 @@
+# Build library as source distribution with included model files
+
+# Either, call `./build.sh` followed by `pip install build/dist/*`
+# Or directly install from source via `pip install -e .`
+
+BUILD_DIR="build"
+CODE_DIR="benchmark_models_petab"
+DATA_DIR="Benchmark-Models"
+
+# Create build folder
+if [ -e $BUILD_DIR ]; then
+    echo "Remove build folder $BUILD_DIR"
+    exit 1
+fi
+mkdir $BUILD_DIR
+
+# Copy code
+cp -r \
+  $CODE_DIR "setup.py" "MANIFEST.in" "../../LICENSE" "../../README.md" \
```

Comment:
Assumes it's running from `src/python/`. Please change all paths to absolute paths, based on the script location.